set-vars.md

```diff
@@ -0,0 +1,171 @@
+---
+title: SET (session settings)
+summary: The SET statement modifies the current settings for the client session.
+toc: false
+---
+
+The `SET` [statement](sql-statements.html) can modify one of the
+session setting variables. These can also be queried via [`SHOW`](show-vars.html).
+
+{{site.data.alerts.callout_danger}}In some cases, client drivers can drop and restart the connection to the server. When this happens, any session configurations made with <code>SET</code> statements are lost. It is therefore more reliable to configure the session in the client's connection string. For examples in different languages, see the <a href="build-an-app-with-cockroachdb.html">Build an App with CockroachDB</a> tutorials.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to modify the session settings.
+
+## Synopsis
+
+General syntax:
+
+{% include sql/diagrams/set_var.html %}
+{% include sql/diagrams/set_time_zone.html %}
```

Comment:
@knz, these diagrams aren't rendered correctly. I've pulled down your branch to figure it out. Here's what you need to do:

1. Insert an empty line between the 2 includes.
2. Regenerate the `set_var.html` image. `cd generate` and then `go run main.go --filter set_var`.

Then view locally to make sure the diagrams look like this:

![screen shot 2017-06-13 at 2 19 18 pm](https://user-images.githubusercontent.com/15893490/27097942-de58bb18-5043-11e7-9d80-ef10d0554101.png)